packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -47,6 +41,21 @@ export class Arc extends React.Component<IArcProps, IArcState> {
     const endAngle = (this.props.data?.endAngle ?? 0) - startAngle;
     const cornerRadius = this.props.roundCorners ? 3 : 0;
 
+    const classNames = getClassNames(getStyles, {
+      // solid color or gradient
+      fill: this.props.enableGradient
```

Comment:
why is fill using redundant conditions. You can define the final value of fill and use it in styles. No need to pass enableGradient property.

Reply:
yea, you're right, I'll make that change

Reply:
I updated it to have `solidFill` and `gradientFill` properties. I need to keep separate fill properties, because the underlying arc needs to be transparent while the gradient enabled arc is visible, otherwise when you hover over the legends you will see the non-gradient arc underneath.

Reply:
why do you need 2 arcs rendered at the same time.

Reply:
I tried using one arc, but the pointer events would not work properly on the arc wrapped in `<foreignObject>` so I kept the original arc to accept pointer events and hid it when gradient is enabled.

Reply:
ok. Lets proceed with this workaround then.